### PR TITLE
[KOGITO-1011] - Github Jenkins Integration for PR

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/KOGITO-1011

Signed-off-by: Mauricio Magnani <mmagnani@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by msmagnanijr** [comment]
I had to make some adjustments.

File: Jenkinsfile [hunk 1]
```diff
@@ -18,21 +18,17 @@ pipeline {
...
         }
-        stage('Clone repository') {
-            steps {
-              sh "mkdir -p ${WORKING_DIR}"
-              sh "git clone https://github.com/kiegroup/kogito-cloud-operator.git ${WORKING_DIR}"
-            }    
-        }
         stage('Initialize') {
             steps {
+                sh "mkdir -p ${WORKING_DIR} && cd ${WORKSPACE} && cp -Rap * ${WORKING_DIR}"
```
Comment (sutaakar) [inline Jenkinsfile]: What is stored in WORKSPACE?

**Comment by ricardozanini** [comment]
Please check the PR requirements. :)

File: Jenkinsfile (hunk 1 above)
Comment (msmagnanijr) [reply Jenkinsfile]: It is the source code of the kogito operator. By default, jenkins clones this to "WORKSPACE" but  for the build to work it needs to be in that specific path. So I'm copying the code to the go path.

Comment (sutaakar) [reply Jenkinsfile]: Ok, thanks

**Pull request merged**